tsconfig.json

```diff
@@ -14,7 +14,8 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react-jsx"
+    "jsx": "react-jsx",
+    "baseUrl": "src"
```

Comment:
is this for the imports?

Reply:
Yes this is to allow absolute imports starting at src